src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.tsx

```diff
@@ -166,7 +162,27 @@ export class DataSourceAggregatedView extends React.Component<
           panelPaddingSize="none"
           anchorPosition="downLeft"
         >
-          <EuiContextMenu initialPanelId={0} panels={panels} />
+          <EuiContextMenuPanel title="Data sources">
+            <EuiPanel color="transparent" paddingSize="s" style={{ width: '300px' }}>
+              <EuiSpacer size="s" />
+              <EuiSelectable
+                options={items}
+                data-test-subj={'dataSourceSelectable'}
+                renderOption={(option) => (
+                  <EuiFlexGroup alignItems="center">
```

Comment:
looks like we are duplicating ourselves now, found https://github.com/opensearch-project/OpenSearch-Dashboards/blob/b4130aa2c1c80033d1bec4d18d44e20612ac0784/src/plugins/data_source_management/public/components/data_source_selector/data_source_selector.tsx#L176